lib/fastlane/plugin/changelog/actions/stamp_changelog.rb

```diff
@@ -57,37 +58,77 @@ def self.stamp(changelog_path, section_identifier, stamp_date, git_tag, placehol
             end
 
             # Output read line
-            file_content.concat(line)
+            file_content.concat(last_line)
+            last_line = line
           end
         end
 
         # 3. Create link to git tags diff
         if !git_tag.nil? && !git_tag.empty?
-          last_line = file_content.lines.last
+          git = ''
           previous_tag = ""
           previous_previous_tag = ""
 
-          if last_line.include? 'https://github.com' # GitHub uses compare/olderTag...newerTag structure
-            previous_previous_tag = %r{(?<=compare\/)(.*)?(?=\.{3})}.match(last_line)
-            previous_tag = /(?<=\.{3})(.*)?/.match(last_line)
+          if last_line.include? 'https://github.com' or last_line.include? 'http://gitlab.' or last_line.include? 'https://gitlab.com' # GitHub and Gitlab use compare/olderTag...newerTag structure
+            git = 'github/gitlab'
```

Comment:
#Did you consider declaring this string as constant [at the top](https://github.com/pajapro/fastlane-plugin-changelog/blob/707376eb6e4db50daba1f7b47ea0292591e97660/lib/fastlane/plugin/changelog/actions/stamp_changelog.rb#L5)? Since it's used later on for comparison, it's kinda error prone.

Reply:
Great idea, this is done ;)